mzLib/mzLib.nuspec

```diff
@@ -2,7 +2,7 @@
 <package>
   <metadata>
     <id>mzLib</id>
-    <version>0.0.1</version>
+    <version>1.10.100</version>
     <title>mzLib</title>
     <authors>Stef S.</authors>
```

Comment:
You'll need to remove/change the `Proteomics` lines in the nuspec

Reply:
Changed